### DISCUSSION TITLE
Video.js support

### DISCUSSION THREAD

**Discussion by DeMiro5001** [LycheeOrg/Lychee#2789] [Ideas]
### Enhancement

I have some mp4 videos with chapters. Playing the videos on lychee doesn't show the chapters so I searched for a way to play them with support for chapters.
Video.js supports it.
Is there way to use it for video playback on lychee?

**Comment by ildyria** [comment]
That would have been helpful if you had gave us a link towards their repo too.

It seems to be quite easy to install and integrate: https://github.com/videojs/video.js

For now I am going to leave it as `good first issue` with the following link.
https://videojs.com/guides/vue/

The files that need changing is : https://github.com/LycheeOrg/Lychee/blob/master/resources/js/views/gallery-panels/Photo.vue#L17
